requirements.txt

```diff
@@ -1,11 +1,27 @@
-numpy>=1.7.0
-scipy>=0.15.1
-pandas>=0.17.0
-scikit-learn>=0.15.2
-biopython>=1.65
-cogent>=1.5.1
-git+https://github.com/biocore/tax2tree.git@63b220f115633ceb028ad7c96b04f9f494cf6ef7
+# this file is order-dependent! see bin/pipdeptree2requirements.py
+
+# to add new dependencies:
+# - create a virtualenv
+# - install one or more packages
+# - install pipdeptree
+# - run `pipdeptree -f --nowarn | bin/pipdeptree2requirements.py` to generate a new requirements.txt
+# - manually replace any references to github repos
+
+six==1.10.0
+numpy==1.11.0
+python-dateutil==2.5.2
+pytz==2016.3
+SQLAlchemy==1.0.12
+biopython==1.66
+decorator==4.0.9
+pandas==0.18.0
+cogent==1.5.3
+futures==3.0.5
 git+https://github.com/cmccoy/peasel.git@b1d5783d8d6d56cb0274e7977d093583c9c5f968
+retrying==1.3.3
+scipy==0.17.0
+scikit-learn==0.17.1
+git+https://github.com/biocore/tax2tree.git@63b220f115633ceb028ad7c96b04f9f494cf6ef7
 git+https://github.com/fhcrc/taxtastic.git@v0.5.7
-futures>=2.0
-retrying
+wheel==0.29.0
+wsgiref==0.1.2
```

Comment:
Do we have to pin all these versions or can we do a >=?  It makes it a little difficult to maintain with other environment software with the same dependencies but slightly different versions.